core/src/components/table/table-component-batch-actions.stories.tsx

```diff
@@ -179,7 +180,7 @@ const BatchActionTemplate = ({
                 column4Width ? `style="width: ${column4Width};"` : ''
               } column-key='mileage' column-title='Mileage' text-align='right'></tds-header-cell>
           </tds-table-header>
-          <tds-table-body enable-dummy-data>
```

Comment:
=/ so we do support setting data as a string. 
Should we create a ticket to change this?

Reply:
Yes, I think it can be done the same way it was done for dropdown options instead...

Reply:
Added an update in the last commit. The only downside that this doesn't work in the docs tab, since storybook can't run js in there.